src/components/routers/WebhookRouter.tsx

```diff
@@ -356,20 +353,17 @@ export default class WebhookForm extends React.Component<WebhookRouterFormProps,
                 }
             }
         }
-
         var newAction: CallWebhook = {
-            uuid: this.props.getActionUUID(),
+            uuid: this.props.action.uuid,
```

Comment:
Without looking, I'm guessing we've now guaranteed that action.uuid is there whereby the function was conditionally determining it?

Reply:
that's exactly right, the `getAction` method on `NodeEditor` is guaranteeing that now